code/modules/food&drinks/food/snacks.dm

```diff
@@ -127,6 +127,11 @@
 
 
 /obj/item/weapon/reagent_containers/food/snacks/attackby(obj/item/weapon/W, mob/user)
+	if(istype(W,/obj/item/weapon/pen))
+		var/n_name = copytext(sanitize(input(usr, "What would you like to name this dish?", "Food Renaming", null)  as text), 1, MAX_NAME_LEN)
```

Comment:
You can just do `var/n_name = stripped_input(user,"What would you like to name this dish?","Food Renaming","",MAX_NAME_LEN)`